examples/manualTestEnv/multiHost/README.md

```diff
@@ -0,0 +1,36 @@
+# multiHost
+
+TiDB, PD, TiKV, TiFlash each in different hosts.
+
+## Usage
+
+1. Start the box:
+
+   ```bash
+   vagrant up
```

Comment:
I'd prefer reusing the `docker/up.sh` utilities to use docker-compose instead of introducing vagrant to the repo. Or even better, we find some way to make the tests independent from any virtualization.